test/integration/testEncrypt.js

```diff
@@ -0,0 +1,82 @@
+const assert = require('assert');
+const path = require('path');
+const os = require('os');
+const fs = require('fs/promises');
+const SnowflakeEncryptionUtil = require('../../lib/file_transfer_agent/encrypt_util').EncryptUtil;
+
+const BASE64 = 'base64';
+
+describe('Test Encryption/Decryption', function () {
+  let encryptUtil;
+
+  before(function () {
+    encryptUtil = new SnowflakeEncryptionUtil();
+  });
+
+  it('GCM - Encrypt and decrypt raw data', function () {
+    const data = 'abc';
+    const iv = Buffer.from('ab1234567890');
+    const key = Buffer.from('1234567890abcdef');
+
+    const encryptedData = encryptUtil.encryptGCM(data, key, iv, null);
+    assert.strictEqual(encryptedData.toString('base64'), 'iG+lT4o27hkzj3kblYRzQikLVQ==');
```

Comment:
we have const for base64 - should we use it?

Reply:
done